src/WebApi.OutputCache.V2/CacheOutputAttribute.cs

```diff
@@ -242,12 +255,12 @@ public override void OnActionExecuting(HttpActionContext actionContext)
                         _webApiCache.Add(baseKey, string.Empty, cacheTime.AbsoluteExpiration);
                         _webApiCache.Add(cachekey, content, cacheTime.AbsoluteExpiration, baseKey);
```

Comment:
I agree with all your points. When I read the http spec on 'no-cache', it was light on details on whether it is specific to client side only. Although majority of cases are for client side use, there is nothing  in the spec that precludes using it on the server. 
Perhaps the easiest option then, is to bypass the cache completely and let it expire on it own per the expiration period. I think the additional attribute makes sense and we can use that in combination with the presence or absence of no-cache.sound good?